{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -19,18 +19,13 @@
 # bootstrap script located in ./scripts/bootstrap.
 
 FROM inveniosoftware/centos7-python:3.6
+
 COPY Pipfile Pipfile.lock ./
 RUN pipenv install --deploy --system
 
-COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
 COPY ./invenio.cfg ${INVENIO_INSTANCE_PATH}
 COPY ./templates/ ${INVENIO_INSTANCE_PATH}/templates/
-
-RUN invenio collect --verbose  && \
```

Comment:
Minor: As spoken in the last call th eonly issue I see is that now people cannot use the Dockerfile by itself. However, they can always inherit from it or it will be dealt with whne using the CLI for deployment.

Reply:
True, developers won't be able to use it by itself anymore, but I don't think it's a common usage and they can inherit from it. If it was kept, webpack (npm) install would re-run everytime `invenio-cli containerize` is run which is a worse experience I think.